screencap.py

```diff
@@ -119,46 +134,46 @@ def retranslateUi(self, mainWindow):
         self.actionmvk.setText(_translate("mainWindow", "mvk"))
 
     def clicked_format(self, extension):
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        f.close()
-        f = open('memory_screenshot.txt','w')
-        lst[1] = extension
-        f.write('{0},{1},{2}'.format(lst[0],lst[1],lst[2]))
-        f.close()
+        previous = data['image']['extension']
+        data['image']['extension'] = extension
+        if previous != data['image']['extension']:
+            with open('memory.json', 'w') as f:
+                f.seek(0)
+                f.write(json.dumps(data, indent=2))
+                f.truncate()
 
     def clicked_folder(self):
-        self.screenshot_path = QtWidgets.QFileDialog.getExistingDirectory(None, 'Select Folder','C:/Users')
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        f.close()
-        f = open('memory_screenshot.txt','w')
-        lst[2] = self.screenshot_path
-        f.write('{0},{1},{2}'.format(lst[0],lst[1],lst[2]))
-        f.close()
-
+        previous = data['image']['path']
+        screenshot_path = QtWidgets.QFileDialog.getExistingDirectory(None, 'Select Folder','C:/Users')
+        data['image']['path'] = screenshot_path
+        if previous != data['image']['path']:
+            with open('memory.json', 'w') as f:
+                f.seek(0)
+                f.write(json.dumps(data, indent=2))
+                f.truncate()
+            
     def clicked_sceenshot(self):
-        f = open('memory_screenshot.txt','r')
-        lst = f.read().split(',')
-        i = int(lst[0])+1
-        i = str(i)
-        path  = lst[2] + '/screenshot' + i + '.' + lst[1]
-        f.close()
+        if (data['image']['path']) == " ":
+            self.clicked_folder()
+        now = datetime.now()
+        timeNow = now.strftime("%d-%m-%Y %H.%M.%S")
+        path = data['image']['path'] + '/'+ data['image']['name'] + timeNow + data['image']["extension"]
         mainWindow.showMinimized()
         time.sleep(0.5)
-        print(path)
         screenshot = ImageGrab.grab()
         screenshot.save(path)
         mainWindow.showNormal()
-        f = open('memory_screenshot.txt','w')
-        f.write('{0},{1},{2}'.format(i,lst[1],lst[2]))
-        f.close()
 
 if __name__ == "__main__":
     qt_warnings()
+    if (os.path.exists("memory.json")) == True:
```

Comment:
```suggestion
    if (os.path.exists("memory.json")):
```
There is no need for `== True`, the function returns True or False and `==` operator does the same.